core/src/main/scala/com/pingcap/tispark/TiBatchWrite.scala

```diff
@@ -757,6 +677,37 @@ class TiBatchWrite(@transient val df: DataFrame,
   private def locatePhysicalTable(row: TiRow): Long = {
     tiTableInfo.getId
   }
+
+  private def splitTableRegion(wrappedRowRdd: RDD[WrappedRow]) = {
+    val rowSize = tiTableInfo.getColumnSize
+    val regionSplitNum = (wrappedRowRdd.count() * rowSize) / (96 * 1024 * 1024)
```

Comment:
pls create an issue: `estimate average row byte size to calculate region split number`

Reply:
suggest to change to
```
val averageColumnSizeInByte = 64
val regionSplitNum = (wrappedRowRdd.count() * rowSize * averageColumnSizeInByte) / (96 * 1024 * 1024)
```

Reply:
96 should be configurable?

Reply:
`rowSize` is how big a row is. We already did the estimation.

Reply:
right, `96` should get from `TiKV`

Reply:
`getColumnSize` better change to `getEstimatedRowSizeInByte`

Reply:
`tiTableInfo.getColumnSize` 's result is max size, usually bigger than real data, better to sample data

Reply:
It is ok to do this way. It is no hard if we split more region.

Reply:
> `tiTableInfo.getColumnSize` 's result is max size, usually bigger than real data, we can calculate real data size

I think it is only to estimate the size pessimistically.